### PR TITLE
Unlock Rails 7.2+

### PR DESCRIPTION
**Pull request by adrianthedev**
Folks are starting to get on the Rails versions 7.2 & 8.
We'd love to make that transition lean.

### CODE REVIEW

**Comment by tvongaza** [comment]
Any updates on this?

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,6 @@ group :local_development do
   gem 'guard-rspec'
   gem 'appraisal'
   gem 'rake'
-  gem 'sqlite3'
+  gem 'sqlite3', '~> 1.4'
```
Comment (adrianthedev) [inline Gemfile:10]: Some tests were failing for different reasons and this is required to make them pass

**Comment by alec-c4** [comment]
Hey! Any updates on this PR?

**Comment by seuros** [comment]
@adrianthedev can you squash your commit ? I don't know why the build is not green.

**Comment by seuros** [comment]
@alec-c4 , this do no test 7.2, just unlock the requirements. 

Do you want to open a PR if @adrianthedev don't respond ?

**Comment by alec-c4** [comment]
@seuros no problem, but let's wait for @adrianthedev to answer

**Comment by seuros** [comment]
Sure. Be sure to lock < 8.0.

Breaking changes in major versions is a major headache.

**Comment by adrianthedev** [comment]
I borked the PR while rebasing. Please open another one @seuros

**Pull request closed without merging**